packages/core/src/Select.js

```diff
@@ -30,7 +30,7 @@ const SelectBase = styled.select`
   }
 `
 SelectBase.defaultProps = {
-  fontSize: 1,
+  fontSize: [2, null, 1],
```

Comment:
JW, what is the significance of making `null` the second element here?

Reply:
It means it will also be 2.  I tend to prefer using null instead of 2 for the sake of inheritance.  If we decide we want to use 3 instead of 2 one day we would only have to change it in one place instead of two.

Reply:
iirc it also means that styled-system won't generate the extra media query if it isn't needed.